README.md

```diff
@@ -65,7 +65,7 @@ you interact with a subset of common frameworks.
 This allows you to start serving models saved in these frameworks straight
 away.
 However, it's also possible to **write [custom
-runtimes](./docs/runtimes/custom)**.
```

Comment:
Great spot!

I think this should be pointing to https://github.com/SeldonIO/MLServer/blob/master/docs/runtimes/custom.md instead, which is meant as the "homepage" for custom runtimes.

Reply:
Yep you're right, I linked the examples one instead